doc/release/0.5.0-notes.rst

```diff
@@ -6,10 +6,89 @@ PyWavelets 0.5.0 Release Notes
 
 .. contents::
 
+PyWavelets 0.5.0 is the culmination of 1 year of work.  In addition to
+several new features, substantial refactoring of the underlying C and Cython
+code have been made.
+
+This release requires Python 2.6, 2.7 or 3.3-3.5 and NumPy 1.9.1 or greater.
```

Comment:
Shall we drop 2.6 and 3.3 after this release? The rest of the world (including numpy) already did that. Mentioning that this is the last release supporting 2.6/3.3 makes sense then.